### ISSUE TITLE
Error: unable to build kubernetes objects

### ISSUE THREAD

**Issue by rdziwinski** [pnnl-miscscripts/miscscripts#3]
Hi!
When I try install your helm chart I get below error:
```
rdziwinski@k01-wro-master-01:~$ helm install dhcpd --namespace "dhcpd" pnnl-miscscripts/dhcpd
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(DaemonSet.spec): missing required field "selector" in io.k8s.api.apps.v1.DaemonSetSpec
```

```
rdziwinski@k01-wro-master-01:~$ kubectl version
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T19:18:23Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-13T11:13:49Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
rdziwinski@k01-wro-master-01:~$ helm version
version.BuildInfo{Version:"v3.0.2", GitCommit:"19e47ee3283ae98139d98460de796c1be1e3975f", GitTreeState:"clean", GoVersion:"go1.13.5"}

```
Could you help me? Any idea?

**Comment by kfox1111** [comment]
Hmm... I don't think I tried installing it with helm 3 yet. Is that what you are doing? helm 3 started being more strict with validation.

**Comment by kfox1111** [comment]
This should be fixed in 0.1.4. You should be able to:
```
helm repo update
```

Then install as normal.

It will take the hub.helm.sh site maybe an hour to update.

Please reopen the issue if you run into further problems.
